spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -643,6 +663,81 @@ public void setRetries(Integer retries) {
 
 	}
 
+	public static class Admin {
+
+		private final Ssl ssl = new Ssl();
+
+		/**
+		 * Id to pass to the server when making requests; used for server-side logging.
+		 */
+		private String clientId;
+
+		/**
+		 * Additional producer-specific properties used to configure the client.
+		 */
+		private final Map<String, String> properties = new HashMap<>();
+
+		/**
+		 * When true, the application context will not load if the broker connection
+		 * fails when attempting to provision topics.
+		 */
+		private boolean failIfNoBrokers;
```

Comment:
Can we rephrase that please? and perhaps change the name of the property? Something that indicates a connection check is done on startup or something?

Reply:
I struggled with this too. `activeBrokerRequired` ? `brokerMandatory` ?

Is there anything similar for other components?